.github/workflows/test-sdks-remote.yml

```diff
@@ -20,6 +20,7 @@ jobs:
           - { repo: "dot-net-server-sdk", workflow: "run-tests.yml", ref: "main" }
           - { repo: "eppo-ios-sdk", workflow: "unit-tests.yml", ref: "main" }
           - { repo: "golang-sdk", workflow: "test.yml", ref: "main" }
+          - { repo: "sdk-common-jdk", workflow: "lint-test-sdk.yml", ref: "tp/apibranch" }
```

Comment:
I assume you'll drop the ref when https://github.com/Eppo-exp/sdk-common-jdk/pull/72 is merged